build/RELEASING.md

```diff
@@ -68,7 +68,15 @@ update the Changelog.
 2. Navigate to the
 [**Actions**](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/actions)
 tab on the repo and monitor the "Pack YOUR_PROJECT_NAME" workflow. The last step
-in the workflow is to publish to Nuget.org.
+in the workflow is to publish to Nuget.org and prepare corresponding github
+release page.
 
-3. Validate that the new version (as specified in step1) of the project is
+3. Validate that the new version (as specified in step 1) of the project is
 successfully published to nuget.org under OpenTelemetry owner.
+
+4. Publish a release in GitHub:
```

Comment:
This is additional burden on maintainers and adds time needed for each release. (currently its as simpl as pushing the right tag only.)

Could you remove the draft one, so the release will be published? The contents can point to the changelog.md file, so there is no manual work needed?

Reply:
The "draft' provides some comfort when onboarding to new automation. I took this same approach on another repo, but eventually removed it because of the extra manual step.

@Kielek, it sounds like you already verified the automation in your fork. 
I agree with @cijothomas here, we should aim to do this without "draft".

Reply:
Done in 9865ca16a6eacd3c63037a0ecbf173998d9982a2 and 829fdfd3d2c1bb28072ea6cc3b1a0257da7c30ad.

Results from the simplified workflow you can find under https://github.com/Kielek/opentelemetry-dotnet-contrib/releases
![image](https://user-images.githubusercontent.com/5972917/218046425-f2965a36-1c57-47cf-a0d3-3d3d081aca2a.png)

Unfortunately I do not found any better way to avoid duplication of code for usntable/stable releases.

Reply:
> Unfortunately I do not found any better way to avoid duplication of code for usntable/stable releases

Okay with the current one. we can try this and improve, as needed.